packages/astro/vendor/vite/dist/node/chunks/dep-35df7f96.js

```diff
@@ -21179,7 +21179,7 @@ function htmlInlineScriptProxyPlugin(config) {
             if (proxyMatch) {
                 const index = Number(proxyMatch[1]);
                 const file = cleanUrl(id);
-                const url = file.replace(config.root, '');
+                const url = file.replace(normalizePath$4(config.root), '');
```

Comment:
This was a 3rd Vite fix that technically wasn’t needed for our fix, but I’m almost positive is needed elsewhere and can’t hurt because it’s the same action.